presto-main/src/main/java/com/facebook/presto/metadata/InMemoryNodeManager.java

```diff
@@ -132,7 +131,7 @@ public InternalNode getCurrentNode()
     public Set<InternalNode> getResourceManagers()
     {
         // always use localNode as resource manager
```

Comment:
- The comment is out of date
- Do we need to completely remove localNode given we have dedicated RM tag now?